scripts/generators/check.js

```diff
@@ -0,0 +1,24 @@
+import log from "npmlog";
+import findAllAfter from "unist-util-find-all-after";
```

Comment:
This is your code... I didn't touch that to be sincere.

Reply:
I 've realized it too late (it was in julien's code too :D) , thus I think comment still apply

Reply:
If I do understand the original code, it was checking all the empty `children` prop for all the first-level sections following the first one. I tried to write [that](https://github.com/SocialGouv/kali-data/pull/96/commits/b571850a0a13e9851c236944bc5e5333801dfe9b) instead and it passes but it's not exactly the same script since find look into all levels and not only the first one.

By the way I noticed that there is at least one empty `children` prop when I also look for the sections included in the first root tree section (agreement main text).